sections/semantics-textlevel.include

```diff
@@ -2993,22 +2993,41 @@
       This example shows how the <{bdo}> element can be used to alter directionality formatting:
 
       <xmp highlight="html">
-        <bdi><bdo dir="rtl">English is a left-to-right language.</bdo></bdi>
+        <bdo dir="rtl">English is a left-to-right language.</bdo>
       </xmp>
 
-      <bdi><bdo dir="rtl">English is a left-to-right language.</bdo></bdi>
+      <bdo dir="rtl">English is a left-to-right language.</bdo>
 
-      This effect is achieved by applying the <code>rtl</code> attribute to the <{bdo}> element.
+      This effect is achieved by applying the <code>dir</code> attribute (with a value of 'rtl' indicating a reading from right to left direction) to the text inside the <{bdo}> element.
+  </div>
+
+  <p class="note">
+    The effect of using the <{bdo}> element is to <em>completely disable</em> the bidirectional algorithm. This means that characters within the <{bdo}> element will be displayed exactly as stored in memory and in the direction specified.
+    This is different than the <{bdi}> element, where the characters will be re-ordered and re-arranged according to the directional algorithm. In most cases, you will probably want to use <{bdi}> element. Use the <{bdo}> element only if you specifically want to disable the bidirectional algorithm completely for that text.
+  </p>
+
+  <div class="example">
+  The following example shows the difference in rendering between the <{bdi}> and <{bdo}> element.
+    <xmp highlight="html">
+    <bdi dir="rtl">שלום Majima! מה שלומך היום?</bdi>
+    <bdo dir="rtl">שלום Majima! מה שלומך היום?</bdo>
+    </xmp>
+    <bdi dir="rtl">שלום Majima! מה שלומך היום?</bdi><br>
+    <bdo dir="rtl">שלום Majima! מה שלומך היום?</bdo>
```

Comment:
I suggest to add a second example of the bdo line, with `dir` set to `ltr` (ie. just change the attribute value).  That then really starts to show the impact of directional sequencing.